cmd/chaos-daemon-helper/main.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Chaos Mesh Authors.
```

Comment:
Can we merge this tool to `chaosctl`?

Reply:
We can, but why?

Reply:
Sorry, I misunderstood, I thought this cmd tool would be used directly by the user, if it is included in the chaos-daemon image, it is perfectly fine.